charts/zipkin/templates/tests/test-connection.yaml

```diff
@@ -16,13 +16,31 @@ spec:
       # Make sure zipkin registered in Eureka at startup
       args: [ 'wget -q --spider http://{{ include "zipkin.fullname" . }}:8761/eureka/v2/apps/ZIPKIN' ]
 {{- end }}
+{{ if .Values.global.testDependencies }}
```

Comment:
used a global here as cycling through a list of objects to pick if "frontend" is there, is quite complicated in helm. maps are easier than lists